pkg/kubectl/cmd/rollout/rollout_history.go

```diff
@@ -49,6 +49,9 @@ var (
 func NewCmdRolloutHistory(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	options := &HistoryOptions{}
 
+	validArgs := []string{"deployment"}
```

Comment:
Hm, are these changes helping in fixing completion? If not, then let's just drop them. The `rollout` commands already know their valid resources via the factory:
```
$ kc rollout history no/127.0.0.1
error: no history viewer has been implemented for {"" "Node"}
```
The error message should be cleaned up in https://github.com/kubernetes/kubernetes/pull/31234